Packs/PagerDuty/Integrations/PagerDuty/PagerDuty.py

```diff
@@ -710,6 +745,55 @@ def get_service_keys():
     }
 
 
+def add_responders_to_incident(incident_id, message, user_requests="", escalation_policy_requests="", requestor_id=DEFAULT_REQUESTOR):
+    """Add the responders to an incident"""
+    if RequestorID is None or RequestorID == "":
+        RequestorID = DEFAULT_REQUESTOR
+    url = SERVER_URL + RESPONDER_REQUESTS_SUFFIX.format(IncidentID)
+    body = {
+        'requestor_id': RequestorID,
+        'message': Message,
+        'responder_request_targets': []
+    }
+    for user_id in UserRequests.split(","):
+        body['responder_request_targets'].append({
+            'responder_request': {
+                "id": user_id,
+                "type": 'user_reference'
+            }
+        })
+    for escalation_policy_id in EscalationPolicyRequests:
+        body['responder_request_targets'].append({
+            'responder_request': {
+                "id": escalation_policy_id,
+                "type": 'escalation_policy_reference'
+            }
+        })
+    response = http_request('POST', url, data=body)
+    return extract_responder_request(response)
```

Comment:
I cant understand what is : `RequestorID`, `Message `, `EscalationPolicyRequests`, `UserRequests` ? 
And why you didn’t used no one from arguments: `incident_id, message, user_requests, escalation_policy_requests, requestor_id` ?

Reply:
Sorry! this isn't ready for review yet, I was waiting for my previous one to get merged before finishing the cleanup on this one.

Reply:
I'm in the process of re-writing the args and writing the tests - I initially had done a different style than the rest of the commands for the inputs, so I'm switching it to snake-case.